lms/djangoapps/dashboard/sysadmin_urls.py

```diff
@@ -16,4 +16,6 @@
     url(r'^gitlogs/(?P<course_id>.+)$', sysadmin.GitLogs.as_view(),
         name="gitlogs_detail"),
     url(r'^task_queue/?$', sysadmin.TaskQueue.as_view(), name="sysadmin_task_queue"),
+    url(r'^mgmt_commands/?$', sysadmin.MgmtCommands.as_view(), name="sysadmin_mgmt_commands"),
```

Comment:
Would there be merit to placing these two endpoints under a common root?

So you'd have something like
```python
https://.../management-commands/index
https://.../management-commands/execute
```

Another approach would be to use the same URL for each, but use `GET` for the list and `POST` for the execute.

Thoughts?